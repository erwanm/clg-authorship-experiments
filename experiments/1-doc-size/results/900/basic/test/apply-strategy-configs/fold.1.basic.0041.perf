0.560400	0.747200	0.750000
