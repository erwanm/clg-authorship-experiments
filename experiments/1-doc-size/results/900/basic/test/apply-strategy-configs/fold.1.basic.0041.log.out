apply-config.sh: applying config '/tmp/expe1/900/process/selected-meta-configs/fold.1.strategy-as-meta.basic.0041.model/strategy-configs/fold.1.basic.0041' to cases '/home/moreaue/dump/expe1//results/900/basic/test/apply-strategy-configs/only-cases.tmp.lwXq9o47'; results written to '/home/moreaue/dump/expe1//results/900/basic/test/apply-strategy-configs/fold.1.basic.0041'
apply-config.sh: done.
